tests/conftest.py

```diff
@@ -165,16 +184,40 @@ def upstream_n_distgit(tmpdir):
 
     u = t / "upstream_git"
     shutil.copytree(UPSTREAM, u)
-    git_add_n_commit(u, tag="0.1.0")
+    initiate_git_repo(u, tag="0.1.0")
 
     d = t / "dist_git"
     shutil.copytree(DISTGIT, d)
-    git_add_n_commit(d, push=True, upstream_remote=str(u_remote))
+    initiate_git_repo(d, push=True, upstream_remote=str(u_remote))
     prepare_dist_git_repo(d)
 
     return u, d
 
 
+@pytest.fixture()
+def sourcegit_n_distgit(tmpdir):
+    temp_dir = Path(str(tmpdir))
```

Comment:
or with [tmp_path fixture](https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture)
```python
def sourcegit_n_distgit(tmp_path):
    temp_dir = tmp_path
```

Reply:
Hmm, it does not work on my version (fedora one) of pytest..;-(

But good to know that. Thanks!